packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -90,7 +90,7 @@ contract L1Block is ISemver, IGasToken {
     /// @notice size of historyHashes.
     uint256 public constant HISTORY_SIZE = 8192;
```

Comment:
Should `HISTORY_SIZE` be also internal with public getter?

Reply:
It's used in `L1Block.t.sol` which is accessing `HISTORY_SIZE()` via interface so can't be internal.

Reply:
Done in 89849bb8c5a114411caf6ab7a761dd4f6e1ef842 .